factfinder/main.py

```diff
@@ -188,14 +222,24 @@ def aggregate_vertical(self, df, from_geotype, to_geotype):
         """
         if from_geotype == "tract" and to_geotype == "NTA":
             return tract_to_nta(df)
-        elif from_geotype == "block group" and to_geotype == "cd_fp_500":
-            return block_group_to_cd_fp500(df)
         elif from_geotype == "tract" and to_geotype == "cd":
             return tract_to_cd(df)
+        elif from_geotype == "block group" and to_geotype == "cd_fp_500":
+            return block_group_to_cd_fp500(df)
+        elif from_geotype == "block group" and to_geotype == "cd_fp_100":
+            return block_group_to_cd_fp100(df)
+        elif from_geotype == "block group" and to_geotype == "cd_park_access":
+            return block_group_to_cd_park_access(df)
+        elif from_geotype == "block" and to_geotype == "cd_fp_500":
+            return block_group_to_cd_fp500(df)
```

Comment:
block_group_to_cd_fp500 -> block_to_cd_fp500
for the following two elif statements too